src/main/java/com/revature/chronicle/controller/TicketController.java

```diff
@@ -40,6 +40,7 @@
 	@GetMapping(path="notifications")
 	public ResponseEntity <List<Notification>> getNotifications(HttpServletRequest req){
 		 User user =  (User) req.getAttribute("user");
+		 System.out.println("user is: "+user.getUid());

```

Comment:
Please make sure you remove sysouts.

Reply:
For some reason, I still see the sysouts in my editor. Did you forget to remove some of them? @MichaelPearson-gif @jamesolas